src/resolvers/account/index.ts

```diff
@@ -0,0 +1,45 @@
+import { generateId, EntityType } from '../../functions/generate-binary-id';
+import AccountModel from '../../models/accounts';
+import { encrypt } from '../../functions/secure-data';
+import { generateToken } from '../../functions/create-token';
+import BinaryResolver from '../../schema/scalars/customs/binary-scalar';
+import EmailAddressResolver from '../../schema/scalars/customs/email-scalar';
+
+interface SignUpInput {
```

Comment:
use `types` for plain object definitions, also separate this type into a different folder called `types`